From 5d28e362388fb01d8ad2365d1e917b60bcc43fe1 Mon Sep 17 00:00:00 2001
From: Sam Ravnborg <srn@skov.dk>
Date: Wed, 15 Feb 2017 15:56:38 +0100
Subject: [PATCH] arm9-cpu: fix so we use MCI1 for SD-CARD

Also disable CONFIG_SDRAM_16BIT - it was wrong.
(Do not know what it is really used for, but it was not selected in the
origianl code)
---
 contrib/board/skov/arm9_cpu/arm9_cpu_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
index 2b13541..5cbce37 100644
--- a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
+++ b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
@@ -1,6 +1,6 @@
 CONFIG_ARM9_CPU=y
 CONFIG_RAM_128MB=y
-CONFIG_SDRAM_16BIT=y
 CONFIG_SDCARD=y
+CONFIG_AT91_MCI1=y
 CONFIG_JUMP_ADDR="0x23800000"
 CONFIG_DEBUG=y
-- 
1.8.3.1

